cmd/bfgd/bfgd.go

```diff
@@ -152,6 +152,11 @@ func _main() error {
 	return nil
 }
 
+func init() {
+	version.Component = "bfgd"
+	welcome = "Hemi Bitcoin Finality Governor " + version.BuildInfo()
```

Comment:
Why isn't this done in global var () ?

This is true for the other ones below this as well.